src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,56 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents task of user
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    String dateString;
+    LocalDate date;
```

Comment:
Don't forget the access modifier! 😄 It is important as attributes should be declared private when possible and neglecting access modifier makes the attributes implicitly public.